### PR TITLE
Move to Go 1.15.5

### PR DESCRIPTION
**Pull request by mastersingh24**
#### Type of change

Improvement (currency)

#### Description

Update to Go 1.15.5

### CODE REVIEW

**Comment by btl5037** [comment]
Missed the Makefile: https://github.com/hyperledger/fabric-ca/blob/master/Makefile#L57

**Comment by mastersingh24** [comment]
> Missed the Makefile: https://github.com/hyperledger/fabric-ca/blob/master/Makefile#L57

👎

File: lib/ca_test.go [hunk 1]
```diff
@@ -217,7 +218,7 @@ func testValidMatchingKeys(cert *x509.Certificate, t *testing.T) {
 		t.Error("Should have failed, public key and private key do not match")
 	}
 
-	err = validateMatchingKeys(cert, string(0))
+	err = validateMatchingKeys(cert, strconv.Itoa(0))
```
Comment (sykesm) [inline lib/ca_test.go:221]: Just a comment: Using Itoa to render the number `0` as the string `"0"` could be done by simply replacing it with the literal `"0"`.

It also looks like most of these changes are related to trying to read or create files in bad paths.

Comment (mastersingh24) [reply lib/ca_test.go:221]: >Just a comment: Using Itoa to render the number 0 as the string "0" could be done by simply replacing it with the literal "0"

True enough

>It also looks like most of these changes are related to trying to read or create files in bad paths. I can't tell if the author was initially trying to introduce null terminators into the strings (rune(0) or "\x00") or just come up with a bogus value.

seems like both depending on the test

Comment (mastersingh24) [reply lib/ca_test.go]: I went with bogus strings to make the intention clearer.

**Review by sykesm — APPROVED** [review]
lgtm

**Pull request merged**